test/procedures-shared.test.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2022 Cloudcontainers B.V.
+ */
+
+var test = require('tape').test;
+var shared = require('../lib/procedures/shared.js');
+
+test('nextIdNoInstances', function (t) {
+    var arg = {
+        change: {
+            service: {
+                name: "portolan"
+            }
+        },
+        instances: []
+    };
+
+    arg = shared.getNextInstAliasOrdinal(arg);
+
+    t.equal(arg.nextId, 0);
+
+    t.end();
+});
+
+test('nextIdSingleInstance', function (t) {
+    var arg = {
+        change: {
+            service: {
+                name: "portolan"
+            }
+        },
+        instances: [
+            {
+                params: {
+                    alias: "portolan0"
+                }
+            }
+        ]
+    };
+
+    arg = shared.getNextInstAliasOrdinal(arg);
+
+    t.equal(arg.nextId, 1);
+
+    t.end();
+});
+
+// This would return 8 as "7" sorts after "10"
```

Comment:
This comment feels backward to me. I'd rather it express the desired behavior, rather than the broken behavior.

Something like:

    // Verify alias index is sorted numerically, not as a string.

Reply:
I've clarified the intent of the test, instances should go up to eleven 😁